exercise_4.1.js

```diff
@@ -0,0 +1,10 @@
+
+const cells =
+        document.querySelectorAll('td');
+        cells.forEach(function(cell) {
+            var number = cell.innerText;
+            alert(number + " squared is: " + 
+            Math.pow(number, 2))
+            alert(number + " cubed is: " + 
+            Math.pow(number, 3))
+        })
```

Comment:
I'm horrified by relying on the HTML for data, but I like the use of `const` and the document query selector which we've not gone over yet!